### PR TITLE
JI-5544 — Column: View is scrolled upon clicking start over

### PR DESCRIPTION
**Pull request by Gwenillia**

### CODE REVIEW

File: src/scripts/questionnaire.js [hunk 1]
```diff
@@ -422,7 +422,10 @@ export default class Questionnaire extends H5P.EventDispatcher {
         }
       });
 
-      this.move(0, true);
+      if (H5P.instances[0].libraryInfo.machineName === "H5P.Questionnaire") {
```
Comment (Gwenillia) [inline src/scripts/questionnaire.js:425]: inheritance is not supported for es6+ with h5p core. Have to use MachineName

Comment (otacke) [reply src/scripts/questionnaire.js:425]: It should. Try adding `["@babel/plugin-transform-classes", { "loose": true }]` to Babel's config (cmp. https://babeljs.io/docs/babel-plugin-transform-classes)

But querying `H5P.instances` from within a content type feels a little odd anyway. Guess there's no better way though.

Comment (Gwenillia) [reply src/scripts/questionnaire.js]: Nice one :) I totally ignore about this babel stuff and ecma 5 ahah. Thanks it seems working as expected

Comment (Gwenillia) [reply src/scripts/questionnaire.js]: 1 Well moving to the first question does the job yup
2 Mh which would not actually change the current behaviour then ? In this case we'd like to make it optimal for h5p.com but indeed would be nice if we could make it optimal everywhere. Guess we have to start somewhere and see how it goes before ahah But I'll adress your point to PO, nice one :) 
3. I'm not querying H5P.instances anymore in this case 🤔

Comment (Gwenillia) [reply src/scripts/questionnaire.js]: Edit : Just thinking about point 2. I'm not sure it would happen anyway with the current behaviour :)

Comment (otacke) [reply src/scripts/questionnaire.js]: 1. But you're not only skipping the focus, but also skipping moving to the first question, don't you? Resetting should not move to the first question? Can well be wanted that way, but it is not what I would have expected.
2. Example: Have a WordPress page with lots of stuff in it and a Questionnaire instance at the bottom. Reset that instance (someone might create a plugin to do similar things in the H5P WordPress integration or whatever - the question type contract is an API for people to use for those kind of things). The page will scroll the page to the instance on focus. But even without that example: Why should the reset behavior differ depending on whether the instance is subcontent or not?

**Pull request merged**